src/test/api/karate/cards/postCardsEntitiesToRespond.feature

```diff
@@ -59,7 +59,6 @@ Feature: Post cards with entitiesAllowedToRespond and/or entitiesRequiredToRespo
     Then status 201
     And match response.count == 1
 
-  Scenario: Push cards non-compliant values of entitiesAllowedToRespond and entitiesRequiredToRespond
```

Comment:
We should also remove the "with compliant values" in the remaining scenario. 

I was also going to say that we don't need the karate tests anymore now that there's no constraint (see comment above), but then they're handy to test the behaviour of the header in card details in the UI, so maybe we can keep them. What do you think?

Reply:
I propose to not remove these karate tests. They are already written and they help to avoid regression.

Reply:
True, they're already written.